pgo/cmd/create.go

```diff
@@ -291,6 +308,11 @@ func init() {
 		`The size of the PVC capacity for primary and replica PostgreSQL instances. Overrides the value set in the storage class. Must follow the standard Kubernetes format, e.g. "10.1Gi"`)
 	createClusterCmd.Flags().StringVarP(&BackrestPVCSize, "pgbackrest-pvc-size", "", "",
 		`The size of the PVC capacity for the pgBackRest repository. Overrides the value set in the storage class. This is ignored if the storage type of "local" is not used. Must follow the standard Kubernetes format, e.g. "10.1Gi"`)
+	createClusterCmd.Flags().StringVarP(&BackrestRepoPath, "pgbackrest-repo-path", "", "",
+		"The pgBackRest repository path that should be utilized instead of the default. Required "+
+			"for standby\nclusters to define the location of an existing pgBackRest repository.")
+	createClusterCmd.Flags().BoolVarP(&Standby, "standby", "", false, "Creates a standby cluster "+
+		"that replicates from a pgBackRest repository in AWS S3.")
 
```

Comment:
In #1342 I also alphabetized all these flags. Enjoy the rebase!

Reply:
Following a rebase, these flags are now properly alphabetized with the others.